media-libs/dav1d/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<use>
```

Comment:
Do you want to maintain this package?

Reply:
I'm not sure. I'm not that closely in touch with dav1d development, so I don't know if I'll be able to stay on top of new releases in a timely manner. If nobody else better qualified can step up to the task, I could do it.